Formula/h/hackernews-tui.rb

```diff
@@ -0,0 +1,23 @@
+class HackernewsTui < Formula
+  desc "Terminal UI to browse Hacker News"
+  homepage "https://github.com/aome510/hackernews-TUI"
+  url "https://github.com/aome510/hackernews-TUI/archive/refs/tags/v0.13.4.tar.gz"
+  sha256 "77abc579f58ed33905f00eb4d6047b1d7f30a0b4ed98b44b36ff95648c6ebae0"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
```suggestion

```

Reply:
Unless upstream uses GitHub pre-releases, we don't need to specify this. It'll check the git tags automatically.

Reply:
All versions before v0.2.1 were pre-release. There hasn't been another since, so I don't mind removing this.

Reply:
I was waiting for all the discussions to culminate so that I could make one single commit. Is it better to make commits as you go to resolve discussions and squash everything at the end?

Reply:
@manakbisht Either is fine!